bench/db_bench.ml

```diff
@@ -77,6 +77,14 @@ module Index = struct
 
   let root = "_bench" // "db_bench"
 
+  let names =
+    let c = ref 0 in
+    fun object_type ->
```

Comment:
AFAICS, the arguments to this function aren't really `object_type`s. Could we use it as the object name directly? (rather than creating less-useful `index_#` names)